mod/homepage/set/self/homepage_numbers.rb

```diff
@@ -22,6 +22,5 @@ def featured_answers
 
   def count card_type
     number = nest card_type, view: :count
```

Comment:
remove `number` as it is not used anymore

Reply:
`def count card_type
    nest card_type, view: :count
 end`

Yes?